lib/components/ms-input/MsAddressInput.vue

```diff
@@ -54,8 +55,20 @@ defineExpose({
 const geoapifyApi = new GeoapifyAPI(props.geoapifyApiKey);
 const addressesFound = ref<Array<Address>>([]);
 const address = ref('');
+const lostFocus = ref(true);
```

Comment:
`lostFocus` is weird for a variable name. You initialize it to true, but it's incorrect: focus has not been lost yet.
Why not rename it to `isFocused` instead? Init it to false, set it to false in `handleFocusLost`, to true in `handleFocus`, and the condition become `v-if="addressesFound.length > 0 && isFocused`.